### PR TITLE
Display all values with outlineproperties

### LINKED ISSUE #599: [Outline] Results with format=outline and outlineproperties displays each page only one time
### Setup

- MW version: 1.34.1
- DB (MySQL etc.): MySQL 5.7
- PHP version: 7.2
- SMW version: 3.1.6
- SRF version: 3.1.0
- Browsers and versions (if applicable): not applicable

### Issue

For the format "outline" with the option "outlineproperties", when a page returned by the query has multiple values for the given property, it is only displayed in the first value.

**Steps to reproduce**

For instance we have the following query:

```
{{#ask:[[Category:Article]]
|?Topic
|format=outline
|outlineproperties=Topic
}}
```

and the page "My wonderful article" contains:

```
[[Topic::My first topic]]
[[Topic::My second topic]]
[[Category:Article]]
```

**Expected result:**

The result of the query should be:

My first topic
- My wonderful article

My second topic
- My wonderful article

**Observed result:**

Currently the result of the query is:

My first topic
- My wonderful article

See [this page on sandbox.smw](https://sandbox.semantic-mediawiki.org/wiki/Query_outline_with_outlineproperties_with_multiple_values).

### PR DESCRIPTION
**Pull request by Seb35**
This PR is made in reference to: #599

This PR includes:
- [x] Update of RELEASE-NOTES.md
- [x] Tests (unit/integration)
- [x] CI build passed

Fixes #599

### CODE REVIEW

**Comment by Seb35** [comment]
I will improve this PR in the following days to check the various code quality requirements.

**Comment by Seb35** [comment]
@JeroenDeDauw @gesinn-it-gea: the CI now passes on this PR, could you review it? (I just rebased it)

File: tests/phpunit/Integration/JSONScript/TestCases/outline-02.json [hunk 1]
```diff
@@ -87,6 +95,16 @@
...
 				]
 			}
+		},
+		{
+			"type": "parser",
+			"about": "#2 (`outlineproperties=Assigned to` with a same page assigned to two people)",
+			"subject": "SRF/Outline/Q.3",
+			"assert-output": {
+				"not-contain": [
+					"<li class=\"mw-empty-elt\"></li>"
```
Comment (JeroenDeDauw) [inline tests/phpunit/Integration/JSONScript/TestCases/outline-02.json:105]: Not the most safe assertion. This test could easily not be doing what is intended, or stop doing that, while still passing. 

Is it possible to instead check that the two values are both present?

Comment (Seb35) [reply tests/phpunit/Integration/JSONScript/TestCases/outline-02.json:119]: I added the check of the two values.

**Pull request merged**